packages/workbox-precaching/test/sw/revisioned-caching.js

```diff
@@ -118,48 +118,6 @@ describe('sw/revisioned-caching()', function() {
     });
   });
 
-  const invalidTypeRevisions = [
-    null,
-    undefined,
-    false,
-    true,
-    12345,
-    {},
-    [],
-  ];
-  invalidTypeRevisions.forEach((invalidRevision) => {
-    it(`should throw an errror for bad revision value '${JSON.stringify(invalidRevision)}'`, function() {
```

Comment:
It was due to https://github.com/GoogleChrome/workbox/pull/664#discussion_r126388087, but I can reinstate it.